contrib/pipelines/secadvisor/README.md

```diff
@@ -1,34 +1,85 @@
 # Security Advisor Pipeline
 
-Implements the IBM Security Advisor pipeline
+The IBM Security Advisor pipeline extracts flow data from skydive, filters out
```

Comment:
Maybe start with one line ... and than provide all 'details' so ... start with 

The IBM Security Advisor pipeline export flows from Skydive into IBM Object Storage (COS).

And than the details ...

Reply:
COS is going to be addressed in the blog, so i prefer to stick only with what is needed to get the test scenario working

Reply:
ok ... so ... when the blog will be ready I propose that you link from here to the blog (and vice versa)

Reply:
@KalmanMeth - please reference the README from the blog - as to visa versa - I rather not reference out to external resources